composer.json

```diff
@@ -7,6 +7,10 @@
         {
             "name": "Taylor Otwell",
             "email": "taylorotwell@gmail.com"
+        },
+        {
+            "name": "Tom de Goede",
+            "email": "tom@allesonline.eu"
```

Comment:
If you want this PR to have even the slightest chance of surviving, you should remove this.